blue-hat/ena/hat.go

```diff
@@ -0,0 +1,49 @@
+package main
+
+import (
+	"image"
+
+	"image/color"
+	"image/png"
+
+	// registration of the png format
+	_ "image/png"
+	"os"
+)
+
+func main() {
+
+	const sourceFile = "red_hat.png"
+	const outputFile = "modified_hat.png"
+
+	fin, err := os.Open(sourceFile)
+	defer func() { fin.Close() }()
+	if err != nil {
+		panic(err)
```

Comment:
This prints the stack trace. Not sure this is what you want.

Reply:
I know what it does.